pkg/api/v20220904/clustermanager_convert.go

```diff
@@ -0,0 +1,88 @@
+package v20220904
+
+import (
+	"encoding/json"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
```

Comment:
wrong line.  our linter would complain

Reply:
PR is green so linter probably broken sometime back.  lol

Reply:
@facchettos did our MSFT hack script/golang program used to ensure the license was on a specific line?  I know some changes were made a few months back but it doesn't seem to care now.

Reply:
after a quick check, I haven't modified the hack file, it was just renamed. The code was (and still is):
 ```go
		if !bytes.Contains(b, goLicense) {
			i := bytes.Index(b, []byte("package "))
			i += bytes.Index(b[i:], []byte("\n"))

			var bb []byte
			bb = append(bb, b[:i]...)
			bb = append(bb, []byte("\n\n")...)
			bb = append(bb, goLicense...)
			bb = append(bb, b[i:]...)

			err = ioutil.WriteFile(path, bb, 0666)
			if err != nil {
				return err
			}
		}
```

So it adds it to a specific line, but if at any place in the file there is the license, it won't change it, so the linting will pass